server/pulp/server/managers/status.py

```diff
@@ -55,6 +58,7 @@ def get_broker_conn_status():
         conn.connect()
         conn.release()
         return {'connected': True}
-    except:
+    except Exception, e:
```

Comment:
Just to make the py3 transition easier when it comes, it's good to start using the ``except Exception as e:`` syntax.